packages/nitro-protocol/contracts/ForceMove.sol

```diff
@@ -84,7 +84,7 @@ contract ForceMove is IForceMove {
         emit ChallengeRegistered(
             channelId,
             largestTurnNum,
-            uint48(now) + fixedPart.challengeDuration, // This could overflow, so don't join a channel with a huge challengeDuration
+            uint48(block.timestamp) + fixedPart.challengeDuration, // This could overflow, so don't join a channel with a huge challengeDuration
```

Comment:
What's the max `challengeDuration`? We could add a check to the wallet

Reply:
I think (2**48 − 1 - now) but I would need to think a bit more to be sure. But I think this is probably the safe lower bound on the maximum.